platform/openide.util.ui/src/org/openide/util/CachedHiDPIIcon.java

```diff
@@ -35,7 +38,7 @@
  * representations for HiDPI displays. Bitmaps for multiple HiDPI scaling factors can be cached at
  * the same time, e.g. for multi-monitor setups. Thread-safe.
  */
```

Comment:
The newly visible class should also get `@since` annotation.

Reply:
I'll add it.